syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -482,45 +486,88 @@ def exp(self, iterations=8):
         """
         return (1 + self / 2 ** iterations) ** (2 ** iterations)
 
-    def sigmoid(self, method="exp"):
+    def sign(self):
+        return (self > 0) + (self < 0) * (-1)
+
+    @staticmethod
+    def _sigmoid_exp(tensor):
         """
-        Approximates the sigmoid function
+        Compute the sigmoid using the exp approximation
+        sigmoid(x) = 1 / (1 + exp(-x))
+        sigmoid(x) = (sigmoid(|x|) - 0.5) * sign(x) + 0.5
```

Comment:
Detail a bit the method for people which don't know about numerical stability etc